src/actions/sources/index.js

```diff
@@ -0,0 +1,3 @@
+export { createPrettySource } from "./createPrettySource";
+export { loadSourceText } from "./loadSourceText";
+export { togglePrettyPrint } from "./togglePrettyPrint";
```

Comment:
Ws seem to be having lots of actions files that seem to have just one action. Wondering if we can group related actions
e.g `createPrettySource` and `togglePrettyPrint` can go into one file `prettyPrint.js`

```js
export { createPrettySource, togglePrettyPrint } from "./prettyPrint";
```

Reply:
good idea. now that we have extracted `togglePrettyPrint` we can consolidate the two!